src/Sarif/Autogenerated/SarifRewritingVisitor.cs

```diff
@@ -594,7 +594,14 @@ public virtual Resources VisitResources(Resources node)
                         var value = node.Rules[key];
                         if (value != null)
                         {
-                            node.Rules[key] = VisitNullChecked(value);
+                            ISarifNode sarifNode = value as ISarifNode;
```

Comment:
>ISarifNode [](start = 28, length = 10)

Don't put this in the `Autogenerated` code. It will be overwritten on the next build, and you've already copied the visitor to `NotYetAutogenerated` anyway.

Reply:
We need this in the autogenerated code. Otherwise, when others sync to this change, the new file will be copied over and everyone will have this delta in their Autogenerated copy. This is just the way things work. :)

---
In reply to: [221318810](https://github.com/Microsoft/sarif-sdk/pull/1103#discussion_r221318810) [](ancestors = 221318810)